src/javascript/app_2/App/Components/Elements/DenialOfServiceModal/denial_of_service.jsx

```diff
@@ -0,0 +1,39 @@
+import React                from 'react';
+import PropTypes            from 'prop-types';
+import { getAccountOfType } from '_common/base/client_base';
+import { localize }         from '_common/localize';
+import URL                  from '_common/url';
+import FullPageModal        from 'App/Components/Elements/FullPageModal/full_page_modal.jsx';
```

Comment:
@easteregg 
what's your idea ?

```suggestion
import FullPageModal        from '../FullPageModal/full_page_modal.jsx';
```

Reply:
They don't need to be relatively close to each other in order to work. so refactoring a relative address is always harder IMO.

Reply:
@easteregg 
Since both of them are wrapped by a supreme package (App) so it's better to use the relative address.

Reply:
I don't think being wrapped by another package is enough of a reason to make them relative. Can you elaborate on what you mean by that? how is that makes the code better?